app/services/beta/search/perform_search_service.rb

```diff
@@ -14,6 +14,7 @@ def call
         path = "/api/beta/search?#{query_params}"
         response = api.get(path)
         parsed = parse_jsonapi(response)
+        merge_meta_to_result!(parsed, response)
```

Comment:
This shouldn't be necessary AFAIK

Reply:
When the parse_jsonapi returns the object there is no meta merged into it. There is a statement to check and merge meta inside the tariff parser api but the meta data does not exist in the `data` part of the response. That is why we need to merge the meta part and i added this line.
<img width="713" alt="image" src="https://user-images.githubusercontent.com/108667911/185886608-61f3c1aa-8313-4e49-b422-53d615303c67.png">

Reply:
I didn't remove the code inside tariff parser api or alter it since there could be edge case(s) regarding that, and that's why i waited for your review.

Reply:
Oh, okay. Probably we should add a parse_top_level_meta! method and add tests for that. I don't think it should affect any of the existing api entity models